FirebasePerformance/Sources/Configurations/FPRConfigurations.h

```diff
@@ -62,6 +62,18 @@ FOUNDATION_EXTERN FPRConfigName kFPRConfigInstrumentationEnabled;
 
 #pragma mark - Configuration fetcher methods.
 
+/**
+ * Controls how the SDK detects and drops prewarmed app start traces:
+ * 0 = drop all app start traces;
+ * 1 = drop only if ActivePrewarm returns true;
+ * 2 = drop only if double dispatch returns true;
+ * 3 = drop if either ActivePrewarm or double dispatch returns true;
+ * 4 = do not drop any app start traces.
+ *
+ * @return an integer representing the detection mode of prewarm app start traces.
+ */
+- (uint32_t)prewarmDetectionMode;
```

Comment:
Can we make this function return the ENUM as well? Typically the enum should be defined in the FPRConfigurations.h and we use those definitions in the whole of SDK. The ENUM definition should not be in the implementation file of FPRAppActivityTracker.m

Reply:
Will move it to `FPRConfigurations.h`